package.json

```
@@ -30,6 +30,7 @@
     "lint:format": "prettier --check .",
     "lint:commit": "commitlint --from=origin/main",
     "lint:ng": "ng lint",
+    "lint:eslint": "eslint playwright",
```

Since we only lint `playwright` and also run eslint as part of `lint:ng`, shall we rename this one? E.g.

```suggestion
    "lint:playwright:eslint": "eslint playwright",
```

```suggestion
    "lint:ng:playwright": "eslint playwright",
```